src/main/java/org/testng/IInvocationStatus.java

```diff
@@ -1,8 +1,12 @@
 package org.testng;
 
 public interface IInvocationStatus {
+
   void markAsInvoked();
```

Comment:
Is it still useful? With invocation time we can now consider that it is invoked when a time is set, right?

Reply:
Not needed anymore. We can just manage with the invocation time.